SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2621,6 +2613,11 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 				ActorSystem->ProcessPositionUpdates();
 			}
 
+			if (ActorSystem.IsValid())
```

Comment:
Worth checking in the case right above here as well I think.

Reply:
Good catch